<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Roles</class>
 <widget class="QMainWindow" name="Roles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Roles</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.228856 rgba(223, 0, 255, 255), stop:0.761194 rgba(0, 186, 224, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(72, 201, 176);</string>
      </property>
      <item>
       <property name="text">
        <string>Metge</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pacient</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(72, 201, 176);</string>
      </property>
      <property name="text">
       <string>Entrar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="usuari">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="usuari_3">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Escull Rol&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
